src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -131,118 +172,74 @@
           null,
         @end
         @if context.messages.filterBundlingMethods(ifaceConfig, context.getNonStreamingMethods(service))
-          bundleDescriptors,
+          BUNDLE_DESCRIPTORS,
         @else
           null,
         @end
-        {'x-goog-api-client': googleApiClient},
-        opts.grpc);
+        {'x-goog-api-client': googleApiClient});
   @end
 @end
 
-@private constructor(documentation, service)
+@private constructor(service)
   @let ifaceConfig = context.getApiConfig.getInterfaceConfig(service), \
-       serviceName = context.getApiWrapperName(service)
+       serviceName = context.getApiWrapperName(service), \
+       documentation = context.defaultComments(service)
     /**
      @if documentation
        {@comments(documentation)}
        *
      @end
      * @@class
-     * @@param {?Object} opts - The optional parameters.
-     * @@param {String} opts.servicePath
+     * @@param {Object=} opts - The optional parameters.
+     * @@param {String=} opts.servicePath
      *   The domain name of the API remote host.
-     * @@param {number} opts.port
+     * @@param {number=} opts.port
      *   The port on which to connect to the remote host.
-     * @@param {Function} opts.getCredentials
-     *   Custom function to obtain the credentials.
-     * @@param {grpc.ClientCredentials} opts.sslCreds
+     * @@param {grpc.ClientCredentials=} opts.sslCreds
      *   A ClientCredentials for use with an SSL-enabled channel.
-     * @@param {Object} opts.grpc
-     *   When specified, this is used as the grpc module. Otherwise
-     *   the grpc package will be loaded from the dependency (typically
-     *   the one within 'google-gax' will be loaded). This will be useful
-     *   to share channels among multiple APIs.
-     * @@param {Object} opts.clientConfig
+     * @@param {Object=} opts.clientConfig
      *   The customized config to build the call settings. See
      *   {@@link gax.constructSettings} for the format.
-     * @@param {number} opts.timeout
+     * @@param {number=} opts.timeout
      *   The default timeout, in seconds, for calls made through this client.
-     * @@param {number} opts.appName
+     * @@param {number=} opts.appName
      *   The codename of the calling service.
-     * @@param {String} opts.appVersion
+     * @@param {String=} opts.appVersion
      *   The version of the calling service.
      */
-    function {@serviceName}(opts) {
+    function {@serviceName}(gaxGrpc, grpcClient, opts) {
```

Comment:
Documentation for those parameters are not helpful actually. Moved the existing parameter documentation to the actual builder function.